pyproject.toml

```diff
@@ -37,6 +37,7 @@ capycli = "capycli.main.cli:main"
 python = "^3.8" # drop support for 3.6 and 3.7 because of wheel and cli-support
 colorama = "^0.4.3"
 requests = "^2.31.0" # fix CVE-2023-32681 
+semver = "^2.9.1"
```

Comment:
Given this is ready for a first review, why did you why you add `semver` here? On a first sight, I couldn't see anything semver related in your changes?

Reply:
PR is in draft status only. Will ask my colleague to review this changes and make it ready for the merge.

Reply:
Hello @gernot-h,
   I have marked as ready for review. 'semver' is used in the below code.
   
https://github.com/sachinshaji/capycli-granularity-download/blob/feat/findsources/capycli/bom/findsources.py#L293